qa/automation/CareerPortal.Tests.Scenarios/2. Extended example/Test/CVforNotOpenPosition.feature

```diff
@@ -0,0 +1,122 @@
+Background: User went to the "Send us your CV" form at the down of the page 
+
+Feature: Send CV for not open job position  
+
+@Happy_pass
+Scenario: Send CV happy pass
```

Comment:
It will be better to have IDs for your Features and Scenarios.

Reply:
Yeees, thanks! I've already understand that ID - must have for all kind of tests (